frontend/src/app/history-query/history-query-detail/history-query-detail.component.ts

```diff
@@ -100,6 +103,8 @@ export class HistoryQueryDetailComponent implements OnInit, OnDestroy {
       .pipe(
         switchMap(params => {
           const paramHistoryQueryId = params.get('historyQueryId') || '';
+          this.historyQueryId = paramHistoryQueryId;
+
```

Comment:
You can directly write `this.historyQueryId = params.get('historyQueryId') || '';` and the `paramHistoryQueryId` variable is not needed anymore.